typings/loader-utils.d.ts

```diff
@@ -0,0 +1,15 @@
+declare module 'loader-utils' {
+  export interface LoaderContext {
+    query: { [key: string]: string | undefined }
+    resourcePath: string
+    context: string
+
+    /** Add file as a dependency to the current module */
```

Comment:
Is this correct or just a copy/paste error?

Reply:
Copy/paste error. Have updated.